components/sst_importer/src/sst_writer.rs

```diff
@@ -61,6 +71,16 @@ impl<E: KvEngine> TxnSSTWriter<E> {
 
         let commit_ts = TimeStamp::new(batch.get_commit_ts());
         for m in batch.get_pairs().iter() {
+            dispatch_api_version!(self.api_version, {
+                let key_mode = API::parse_key_mode(m.get_key());
+                if self.api_version == ApiVersion::V2 && key_mode != KeyMode::Txn {
```

Comment:
Yes, `KeyMode::TiDB` is missed. [Storage::check_api_version](https://github.com/tikv/tikv/blob/8b509a7415e137b3719b1a947d11dfbf20e3bbb5/src/storage/mod.rs#L409) is not quite suitable to this context since the importer is out of the `storage` scope.

Reply:
Yes. So I think it would better to move `check_api_version` to "api_version" crate later.